pkg/component/spec.go

```diff
@@ -81,10 +81,22 @@ func (t *CommandTimeoutSpec) InitDefaults() {
 	t.Stop = 30 * time.Second
 }
 
+// ServiceTimeoutSpec is the timeout specification for subprocess.
+type ServiceTimeoutSpec struct {
+	Checkin time.Duration `config:"checkin" yaml:"checkin"`
+}
+
+// InitDefaults initialized the defaults for the timeouts.
+func (t *ServiceTimeoutSpec) InitDefaults() {
+	t.Checkin = 30 * time.Second
```

Comment:
didn't you say 3 minutes?

Reply:
check-in is always 30 secs default.
see my last comment on this PR:
https://github.com/elastic/elastic-agent/pull/1529#discussion_r1004432188